functests/utils/nodes/nodes.go

```diff
@@ -184,3 +185,18 @@ func HasPreemptRTKernel(node *corev1.Node) error {
 
 	return nil
 }
+
+func BannedCPUs(node corev1.Node) (banned uint64, err error) {
+	cmd := []string{"sed", "-n", "s/^IRQBALANCE_BANNED_CPUS=\\(.*\\)/\\1/p", "/rootfs/etc/sysconfig/irqbalance"}
```

Comment:
so this is a bitmask, right?
this worries me a bit because it suggests it can't hold more than 64 CPUs :\

Reply:
indeed, however the default /etc/sysconfig/irqbalance value suggests a 64bit field. I will dig more to see what is going on, I will increase the field space if necessary. (which probably is)

Reply:
IMO the best option would be the list format of cpuset, like linux does (https://man7.org/linux/man-pages/man7/cpuset.7.html#FORMATS)

Reply:
I Will follow, thanks!

Reply:
sure, even though it seems for irqbalance we are out of luck and stuck with the bitmask :\
```
#
# IRQBALANCE_BANNED_CPUS
# 64 bit bitmask which allows you to indicate which cpu's should
# be skipped when reblancing irqs. Cpu numbers which have their
# corresponding bits set to one in this mask will not have any
# irq's assigned to them on rebalance
#
#IRQBALANCE_BANNED_CPUS=
```

Reply:
OK, I reviewed the irqbalance code and on their side they can handle up until 4096 cpus encoded as bitmask. So irqbalance is fine. The potential issue is on our side (test) if we represent the bitmap using native types, so we are limited by (u)int64.

Reply:
Right, also there is a parsing issue. It turns out they delimit each 32 CPUs with a comma, something like 00000000,00000000,....
Another thing that caught my mind is:
```
From Red Hat Enterprise Linux 7.2, the irqbalance tool automatically avoids IRQs on CPU cores isolated via the isolcpus= kernel parameter if IRQBALANCE_BANNED_CPUS is not set in the /etc/sysconfig/irqbalance file.
```
See https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_for_real_time/7/html/tuning_guide/interrupt_and_process_binding


That means what we are doing actually makes sense, setting the mask to "nothing", which is different from not setting it at all, in which case the irq load balancing will actually be disabled for the isolated cpus.

Reply:
And regarding the CPU mask set, you are perfectly write. I didn't even know about it, thanks! I am going to see how it is done in go.

Reply:
Turns out we are already using kubelet's (!!) cpuset.CPUSet type. Nice, I will use it.